scripts/export_database.pl

```diff
@@ -339,6 +350,15 @@ sub sanitize_field_content {
 					$field_value = sanitize_field_content($field_value, $BAD, "$code barcode -> field $field:");
 				}
 
+				#Obsolete field
+				if ($field eq "obsolete") {
+					$field_value = $product_ref->{obsolete} ? 1 : 0;
+				}
+
+				else {
+					$field_value = ($product_ref->{$field} // "");
+				}
```

Comment:
Why put it in a "else" ?

Normalizing undefined values is a good idea, but it's a bit strange to do it in a else here. Just do it plainly, obsolete field_value is always defined, so it won't change it's value.

Reply:
Updated, thank you!